app/controllers/festivity_location_areas_controller.rb

```diff
@@ -1,7 +1,5 @@
 class FestivityLocationAreasController < ApplicationController
-  include Festivity::Mixins::NotFound
-  no_login_required
-  trusty_layout "normal"
+  include Concerns::FestivityCustomPage
```

Comment:
This concern extracts out some of the boring methods we end up having to put in every custom controller, like setting the layout, adding in the not found mixin, and saying no login is required.